aiohttp/web_fileresponse.py

```diff
@@ -122,29 +132,31 @@ async def _precondition_failed(
         return await super().prepare(request)
 
     def _get_file_path_stat_and_gzip(
-        self, check_for_gzipped_file: bool
-    ) -> Tuple[pathlib.Path, os.stat_result, bool]:
-        """Return the file path, stat result, and gzip status.
+        self, accept_encoding: str
+    ) -> Tuple[pathlib.Path, os.stat_result, Optional[str]]:
```

Comment:
`Optional` is considered confusing, this name doesn't relay the semantics well. There's even a linter forbidding it (it's called `no-optional`). The alternative is to explicitly say that it's one of two types:
```suggestion
    ) -> Tuple[pathlib.Path, os.stat_result, Union[str, None]]:
```
(for py3.10+ it'd be `str | None`, of course)

Reply:
I wouldn't worry about these, they'll get removed in a year or so with pyupgrade regardless.